taxi/views.py

```diff
@@ -36,6 +35,20 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
```

Comment:
The `template_name` attribute is already set to the default value for a ListView when it follows the pattern `app_name/model_name_list.html`. If you are not overriding the default template name, you can remove this line.